service/matching/taskqueue.go

```diff
@@ -111,7 +111,9 @@ func (tn *QualifiedTaskQueueName) GetRoot() string {
 
 // Parent returns the name of the parent task queue
 // input:
-//   degree: Number of children at each level of the tree
+//
+//	degree: Number of children at each level of the tree
+//
```

Comment:
Do you want to revert those comment changes? I don't have strong preference either way. Some changes looks a bit weird though.

Reply:
the formatting is probably due to me using go 1.19